File: Arrays & Hashing/3005. Count Elements With Maximum Frequency/3005.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        map<int,int> freq;
        int maxFreq = 0;
        int res = 0;
        for (int i=0; i<nums.size(); ++i) {
            freq[nums[i]]++;
            maxFreq = std::max(maxFreq, freq[nums[i]]);
        } 
        for (auto itr : freq) {
            if (itr.second == maxFreq) {
                res += maxFreq;
            }
        } return res;
    }
};

File: Arrays & Hashing/3005. Count Elements With Maximum Frequency/3005.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        freq = Counter(nums)
        max_freq = max(v for i,v in freq.items())
        res = 0
        for i,v in freq.items():
            if v == max_freq:
                res += max_freq
        return res